content/news/025/index.md

```diff
@@ -67,6 +67,36 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Sombervale]
+
+![Screenshot of Sombervale depicting the starting location](./sombervale.png)
+
+Sombervale by [@blipjoy] is a game built in 7 days for Rusty Jam. It is styled
```

Comment:
The title and username links don't have a corresponding URL at the bottom of the section.

Reply:
Thank you! Should be resolved now.